partner_event/__init__.py

```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2014 Serv. Tecnol. Avanzados - Pedro M. Baeza
-# © 2015 Antiun Ingenieria S.L. - Javier Iniesta
-# © 2016 Antiun Ingenieria S.L. - Antonio Espinosa
+# © 2016 Tecnativa S.L. - Vicent Cubells
```

Comment:
Usually it's not fun for people if you remove their copyright and add yours.

You may, however, update its format to last template, or in some of this cases (Pedro & Antonio), change Antiun for Tecnativa, but a good rule of thumb is: do not remove others' copyright, as you probably don't want others to remove yours, unless they ask you to. :wink: